gen-app-builder/chat-app/site/src/routes/+page.svelte

```diff
@@ -0,0 +1,112 @@
+      >
+        <blockquote class="mx-auto mb-4 max-w-2xl text-gray-500 dark:text-gray-400">
+          <h3 class="text-lg font-semibold text-gray-900 dark:text-white">Easily ingest data</h3>
+          <p class="my-4 font-light">
+            Including public website data, unstructured data (PDF files), structured data (JSON,
+            BigQuery)
+          </p>
+        </blockquote>
+      </figure>
+      <figure
+        class="flex flex-col justify-center items-center p-8 text-center bg-white rounded-tr-lg border-b border-gray-200 dark:bg-gray-800 dark:border-gray-700"
+      >
+        <blockquote class="mx-auto mb-4 max-w-2xl text-gray-500 dark:text-gray-400">
+          <h3 class="text-lg font-semibold text-gray-900 dark:text-white">Generative AI answers</h3>
+          <p class="my-4 font-light">
+            Retrives indexed documents, then uses large language models to provide natural answers
+            grounded in your data
+          </p>
+        </blockquote>
+      </figure>
+      <figure
+        class="flex flex-col justify-center items-center p-8 text-center bg-white rounded-bl-lg border-b border-gray-200 md:border-b-0 md:border-r dark:bg-gray-800 dark:border-gray-700"
+      >
+        <blockquote class="mx-auto mb-4 max-w-2xl text-gray-500 dark:text-gray-400">
+          <h3 class="text-lg font-semibold text-gray-900 dark:text-white">
+            Multi-channel integrations
+          </h3>
+          <p class="my-4 font-light">
+            Embed a chat widget on your website, connect to a phone gateway or telephony provider,
+            or integrate with other chat and CRM tools
+          </p>
+        </blockquote>
+      </figure>
+      <figure
+        class="flex flex-col justify-center items-center p-8 text-center bg-white rounded-b-lg border-gray-200 md:rounded-br-lg dark:bg-gray-800 dark:border-gray-700"
+      >
+        <blockquote class="mx-auto mb-4 max-w-2xl text-gray-500 dark:text-gray-400">
+          <h3 class="text-lg font-semibold text-gray-900 dark:text-white">Try it yourself</h3>
+          <p class="my-4 font-light">
+            Ask the chatbot on the right questions about products in the Google Store, such as Pixel
+            Phone, Pixel Watch, or Pixel Tablet
+          </p>
+        </blockquote>
+      </figure>
+    </Card>
+  </TabItem>
+  <TabItem class="w-full">
+    <span slot="title">How we built it</span>
+    <p class="text-lg">
+      <b>Step 1:</b> Create a chat app and associated data store in Gen App Builder <br /><br />
+      <b>Step 2:</b> Select the website data type and input your website URLs <br /><br />
+      <b>Step 3:</b> Embed a chat widget into your website and delight your customers!
+    </p>
+  </TabItem>
+</Tabs>
+
+Powered by <a href="https://cloud.google.com/generative-ai-app-builder">Gen App Builder</a> and
+<a href="https://cloud.google.com/dialogflow">Dialogflow CX</a>
```

Comment:
Change the styling for these to indicate that they are links.

Reply:
Will address this in a future beautification PR and make it way better. 😁